packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -313,7 +382,25 @@ export class ChainService implements ChainServiceInterface {
           to: tokenContract.address,
         };
 
-        await this.sendTransaction(increaseAllowanceRequest);
+        const tx = await this.sendTransaction(increaseAllowanceRequest);
+
+        this.logger.info(
+          {
+            assetHolderAddress,
+            tokenAddress,
+            amount,
+            tx: tx.hash,
+          },
+          'Submitted transaction to increase asset holder token allowance'
+        );
+
+        await tx.wait();
```

Comment:
This is a significant change to the chain service API. See my comment on the other transaction mining wait.